### PR TITLE
Ontology manipulation

### PR DESCRIPTION
**Pull request by alkidbaci**
### Added classes for ontology manipulation

- **Ontology representation**: added `OWLOntology_Owlready2` at `owl_ontology.py`
- **Ontology manager**: added `OWLOntologyManager_Owlready2` at `owl_ontology_manager.py`
- **Ontology reasoning**: added `OWLReasoner_Owlready2`, `OWLReasoner_FastInstanceChecker` (FIC), `OWLReasoner_Owlready2_ComplexCEInstances` (CCEI) at `owl_reasoner.py`

### Added more examples
- added `ontology_modification.py` 
- added `ontology_reasoning.py`

### Updated documentation

### CODE REVIEW

File: examples/ontology_modification.py
Comment (Demirrr) [inline examples/ontology_modification.py:1]: Could you rename OWLOntologyManager_Owlready2 as OWLOntologyManager or a like by removing the postfix?

File: examples/ontology_reasoning.py
Comment (Demirrr) [inline examples/ontology_reasoning.py:1]: Could you remove the Owlready2 postfixes?

from owlapy.owl_property import OWLDataProperty, OWLObjectProperty
from owlapy.owl_reasoner import OWLReasoner_Owlready2, OWLReasoner_Owlready2_ComplexCEInstances, BaseReasoner_Owlready2

**Review by Demirrr — CHANGES_REQUESTED** [review]
Would you please remove the postfixes reffering to Owlready2? @alkidbaci

**Comment by alkidbaci** [comment]
**Renamed the following classes:**
- `OWLOntology_Owlready2` &rarr; `Ontogloy` 
- `OWLOntologyManager_Owlready2` &rarr; `OntologyManager`
- `OWLReasoner_Owlready2` &rarr; `OntologyReasoner`
- `OWLReasoner_FastInstanceChecker` &rarr; `FastInstanceCheckerReasoner`
- `OWLReasoner_Owlready2_ComplexCEInstances` &rarr; `SyncReasoner`

PS: Could not do `OWLOntology` or `OWLOntologyManager` because it conflicts with the base class

**Pull request merged**